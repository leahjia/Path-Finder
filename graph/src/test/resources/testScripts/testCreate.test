# DesignMap graph Creation testing file.
# Tests that a graph can be created

# Create a graph
CreateGraph map1
