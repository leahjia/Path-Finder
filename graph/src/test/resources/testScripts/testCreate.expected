# DesignMap graph Creation testing file.
# Tests that a graph can be created

# Create a graph
created graph map1
